public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -36,8 +36,9 @@ define(function(require) {
 
     // Update the UI with a "Saving..." indicator whenever we sync a file
     syncManager.on("file-sync-start", function() {
+		$("#navbar-save-indicator").removeClass("hide");
```

Comment:
You can just make the change and push the changes to your branch. Your PR will update automatically.